packages/patternfly-4/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -29,6 +30,8 @@ export interface ModalContentProps {
   isOpen?: boolean;
   /** Complex header (more than just text), supersedes title for header content */
   header?: React.ReactNode;
+  /** Description */
+  description?: React.ReactNode;
```

Comment:
Does this mean the description can contain markup, too, and doesn't have to just be a string?

Reply:
@mcoker yes.